### PR TITLE
Release Note for ONE 1.20.0

### PR DESCRIPTION
**Pull request by chunseoklee**
- Release Note for ONE 1.20.0

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

File: docs/release/1.20/index.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. ONE documentation master file, created by
+   sphinx-quickstart on Fri Oct 20 15:20:12 2021.
```
Comment (mhs4670go) [inline docs/release/1.20/index.rst:2]:
```suggestion
   sphinx-quickstart on Tue Apr 26 10:18:12 2022.
```

**Comment by glistening** [comment]
LGTM for section about ONE runtime

**Review by llFreetimell — APPROVED** [review]
LGTM (off-the-record opinion : https://github.com/Samsung/ONE/issues/8985#issuecomment-1109214128)

**Review by hseok-oh — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by lemmaa — APPROVED** [review]
LGTM

**Pull request merged**